src/main/java/core/basesyntax/service/impl/AddOperationHandlerImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationHandler;
+
+public class AddOperationHandlerImpl implements OperationHandler {
+    private final FruitDao fruitDao;
+
+    public AddOperationHandlerImpl(FruitDao fruitDao) {
+        this.fruitDao = fruitDao;
+    }
+
+    @Override
+    public void operationWithFruitTransaction(FruitTransaction fruitTransaction) {
+        fruitDao.getAll().merge(
```

Comment:
Okey, so you can create a save method on the dao layer and use method merge under hood